### PR TITLE
Port to jbuilder

### PR DESCRIPTION
**Pull request by mseri**
Let's start small...

When released, this will require changing 
```
%build
oasis setup
make

%install
%{__install} -D -m 755 wsproxy.native %{buildroot}/opt/xensource/libexec/wsproxy
```

to

```
%build
jbuilder build

%install
%{__install} -D -m 755 _build/install/default/bin/wsproxy %{buildroot}/opt/xensource/libexec/wsproxy
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,25 +1,6 @@
...
-
-# ocamlbuild working directory
+# jbuilder working directory
 _build/
 
-# ocamlbuild targets
-*.byte
-*.native
-
 # oasis generated files
```
Comment (gaborigloi) [inline .gitignore]: We don't need this comment anymore

Comment (gaborigloi) [inline .gitignore]: We could maybe change this to `# files generated by jbuilder`

File: wsproxy.opam [hunk 2]
```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+name: "wsproxy"
+maintainer: "xen-api@lists.xen.org"
+authors: [ "Jon Ludlam <jonathan.ludlam@citrix.com>"
+           "Marcello Seri <marcello.seri@citrix.com>" ]
+license: "LGPL-2.0 with OCaml linking exception"
+homepage: "https://github.com/xapi-project/wsproxy"
+dev-repo: "https://github.com/xapi-project/wsproxy.git"
```
Comment (gaborigloi) [inline wsproxy.opam:8]: We could add a `tags: [ "org:xapi-project" ]` line to all our packages. That will be useful in case we upstream them.

**Review by gaborigloi — APPROVED** [review]

File: wsproxy.opam (hunk 2 above)
Comment (mseri) [reply wsproxy.opam:8]: done

**Review by euanh — APPROVED** [review]
s/remomved/removed/ in final changeset's commit message

File: wsproxy.opam [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+bug-reports: "https://github.com/xapi-project/wsproxy/issues"
```
Comment (gaborigloi) [inline wsproxy.opam:9]: Same as for the other PR for `xs-opam`, `build:` field is missing.

**Review by gaborigloi — CHANGES_REQUESTED** [review]
Looks good, the build: field should be added to the opam file

**Pull request merged**